testing_exercise/testing.js

```diff
@@ -0,0 +1,61 @@
+
+
+function replaceWith(stringArr, target, replacement){
+	var newString = "";
+
+	for (var i = 0; i < stringArr.length; i++){
+		
+		if ( stringArr[i] === target ){
+			newString += replacement;
+			stringArr[i] = replacement;
```

Comment:
I don't think this line is doing anything, since `stringArr` is a string, and is therefore immutable.